src/services/types.ts

```diff
@@ -1756,7 +1758,7 @@ export interface Refactor {
     kinds?: string[];
 
     /** Compute the associated code actions */
-    getEditsForAction(context: RefactorContext, actionName: string): RefactorEditInfo | undefined;
+    getEditsForAction(context: RefactorContext, actionName: string, newFile?: string): RefactorEditInfo | undefined;

```

Comment:
Similar reusability comment here—it seems like this should ideally be able to handle arguments for multiple kinds of actions. (This one is internal API though, so it’s less critical to settle on it now.)